File: curso POO uber/Java/UberBlack.java
import java.util.Map;
import java.util.ArrayList;

class UberBlack extends Car{
    Map<String, Map<Integer, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String license, Account driver, 
    Map<String,Map<Integer, Integer>> typeCarAccepted,
     ArrayList<String> seatsMaterial){

        super(license, driver);
        this.typeCarAccepted = typeCarAccepted; 
        this.seatsMaterial = seatsMaterial;
    }
}

File: curso POO uber/Python/UberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterials = []

    def __init__(self, lincense, driver, typeCarAccepted, seatsMaterials):
        super().__init__(lincense, driver)
        
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterials  = seatsMaterials 